build/release.mk

```diff
@@ -29,9 +29,14 @@ release/deps: $(GORELEASER_BIN)
 release/build: release/deps release/clean
 ifeq ($(PRERELEASE), true)
```

Comment:
I still do not understand this... the pipeline is run for prerelease and for release but we push for pre-release only?

Reply:
Pipeline itself is run on `pre-release`, which causes GHA to set `PRERELEASE` to true. However, the `release/build` reciped target is called from another recipe that is run in the push/pr pipeline.

I think this is done to test that packages build in all arches. Rather than just copying the `goreleaser build ...` line, it is called instead. I am not very sure why.